src/pages/SignUp/components/UsernameInput.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
```

Comment:
nemanau kad reikia `UsernameInput`, tiesiog turekime `Input` componenta ir tada jau is virsaus pasakykime kas jis yra, nes db jei kurtume 5 skirtingus inputus, turetume 5 skirtingus componentus, vetoje to kad buti tik vienas

Reply:
Tai tu is virsaus nori padavinet visus tuos type, label,  endAdornment propsus?